### ISSUE TITLE
ORA-00928 - missing SELECT keyword

### ISSUE THREAD

**Issue by scossin** [ohdsi-studies/PioneerWatchfulWaiting#26]
I have the same error in multiple SQL queries due to comments:
```
with primary_events (event_id, person_id, start_date, end_date, op_start_date, op_end_date, visit_occurrence_id) as
(
-- Begin Primary Events
```
If I remove the comment -- Begin Primary Events the error disappears, it's quite mysterious. 

errorReportSql.txt file:
```txt
DBMS:
oracle

Error:
java.sql.SQLSyntaxErrorException: ORA-00928: missing SELECT keyword


SQL:
with primary_events (event_id, person_id, start_date, end_date, op_start_date, op_end_date, visit_occurrence_id) as
(

R version:
R version 4.0.3 (2020-10-10)

Platform:
x86_64-pc-linux-gnu

Attached base packages:
- stats
- graphics
- grDevices
- datasets
- utils
- methods
- base

Other attached packages:
- PioneerWatchfulWaiting (0.3.2)
- DatabaseConnector (4.0.0)
```

**Comment by rfherrerac** [comment]
Same by my side and same error.

DBMS:
redshift

Error:
java.sql.SQLException: [Amazon](500310) Invalid operation: syntax error at end of input 
Position: 118;

SQL:
with primary_events (event_id, person_id, start_date, end_date, op_start_date, op_end_date, visit_occurrence_id) as
(

R version:
R version 4.0.3 (2020-10-10)

Platform:
x86_64-pc-linux-gnu

Attached base packages:
- stats
- graphics
- grDevices
- utils
- datasets
- methods
- base

Other attached packages:
- PioneerWatchfulWaiting (0.3.2)
- DatabaseConnector (4.0.0)

**Comment by keesvanbochove** [comment]
@scossin is this with released version 0.3.2 or with the master branch?

**Comment by scossin** [comment]
It was the latest commit on master branch: https://github.com/ohdsi-studies/PioneerWatchfulWaiting/commit/4cb82b0cd3fb041893c7c842b21c070eea07d38e not the 0.3.2 release

**Comment by keesvanbochove** [comment]
Closed by reverting latest commits to v0.3.2
